handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java

```diff
@@ -34,6 +35,9 @@
  * Triggers an {@link IdleStateEvent} when a {@link Channel} has not performed
  * read, write, or both operation for a while.
  *
+ * The {@link IdleStateHandler} should be placed as first {@link ChannelHandler}
```

Comment:
agreed ... I would say omit it because it is not a requirement to use `IdleStateHandler` as it depends upon the applications desired result. Maybe explaining the behavior it will provide relative to other handlers would be better?